bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -188,6 +188,25 @@ spec:
           containerPort: {{.OVN_NB_PORT}}
         - name: nb-db-raft-port
           containerPort: {{.OVN_NB_RAFT_PORT}}
+        readinessProbe:
+          initalDelaySeconds: 30
+          exec:
+            command:
+            - /bin/bash
+            - -c
+            - |
+              set -e
+              OVN_NODES_ARRAY=({{.OVN_NODES}})
+              AVAILABLE_NODES=0 
+              for node in "${OVN_NODES_ARRAY[@]}"; do
+                node_ip=$(getent ahostsv4 "${node}" | grep RAW | awk '{print $1}')
+                if ovs-appctl -t /var/run/openvswitch/ovnnb_db.ctl  cluster/status OVN_Northbound | grep -q $node_ip; then
```

Comment:
I just tested this, and it turns out we don't ever remove servers if we're not available. So we need to parse the "connections" line. I'll give you a sample in a bit.

Reply:
Yeah, all we need to do is check that the DB port is open. it turns out ovsdb takes care of this for us and only opens the port when consensus is achieved.